src/Apps/SaleAgreements/Components/SaleAgreementsList.tsx

```diff
@@ -17,9 +18,14 @@ export interface SaleAgreementsListProps {
 export const SaleAgreementsList: FC<SaleAgreementsListProps> = ({
   saleAgreements,
 }) => {
+  const sortedSaleAgreements = sortBy(
+    saleAgreements,
+    saleAgreement => saleAgreement?.sale.name
```

Comment:
Totally fine, but another way to do this is the shorthand notation:
```tsx
sortBy(saleAgreements, ['sale.name'])
```

Reply:
Client-side sorting may not work w/ pagination, as in across pages the list may still be out of order. Possibly a candidate for server-side sorting.